components/generic/ContentCard.vue

```diff
@@ -248,7 +247,7 @@
       },
 
       imageNotFound() {
-        this.cardImageUrl = '';
+        this.imageUrl = '';
```

Comment:
Props can not be mutated by the component itself, hence the use of the data variable.